homeassistant/components/advantage_air/select.py

```diff
@@ -19,9 +21,10 @@ async def async_setup_entry(
 
     instance = hass.data[ADVANTAGE_AIR_DOMAIN][config_entry.entry_id]
 
-    entities = []
-    for ac_key in instance["coordinator"].data["aircons"]:
-        entities.append(AdvantageAirMyZone(instance, ac_key))
+    entities: list[SelectEntity] = []
+    if "aircons" in instance["coordinator"].data:
```

Comment:
I think you can use the walrus here.
`if aircons := instance["coordinator"].data.get("aircons"):`

Reply:
Done